examples/mutators/set-label/advanced/README.md

```diff
@@ -0,0 +1,45 @@
+# set-label: Advanced Example
+
+The `set-label` function adds or updates labels in the `.metadata.labels` field
+and other fields that has the same meaning as a label on all resources. You can
+find more details about these fields in the help text of the function.
+
+We use the following `SetLabelConfig` to configure the function.
+
+```yaml
+apiVersion: fn.kpt.dev/v1alpha1
+kind: SetLabelConfig
+metadata:
+  ...
+fieldSpecs:
+  - kind: MyResource
+    group: dev.example.com
+    version: v1
+    create: true
+    path: spec/selector/labels
+labels:
```

Comment:
nit: `labels` should come before `fieldSpecs` as that's what the main user input is.